deploy/crds/performance.openshift.io_performanceprofiles_crd.yaml

```diff
@@ -54,6 +54,11 @@ spec:
                   description: Isolated defines a set of CPUs that will be used to
                     give to application threads the most execution time possible,
                     which means removing as many extraneous tasks off a CPU as possible.
+                    It important to notice, that the CPU manager can choose any CPU
+                    to run the workload, except reserved CPUs, so to guarantee that
+                    your workload will run on the isolated CPU, reserved and isolated
+                    CPUs should include all online CPUs and isolated CPUs field should
+                    be complementary to reserved CPUs.
```

Comment:
I'd rephrase slightly like this:
`your workload will run on the isolated CPU: 1. the union of reserved CPUs and isolated CPUs should include all online CPUs; 2. the isolated CPUs field should be the complementary to reserved CPUs field.`

Reply:
done